docs/root/configuration/other_features/vcl.rst

```diff
@@ -0,0 +1,35 @@
+.. _config_sock_interface_vcl:
+
+VCL Socket Interface
+====================
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.vcl.v3alpha.VclSocketInterface>`
+
+.. attention::
+
+  The VCL socket interface extension is experimental and is currently under active development.
+
+This socket interface extension provides Envoy with high speed L2-L7 user space networking by integrating with `Link fd.io VPP <https://fd.io>`_ through VPP's ``Comms`` Library (VCL).
```

Comment:
`Link` here seems superfluous

Reply:
It is, not sure why/how I added it there. Thanks & fixed!